src/commands/staff/RemoveName.ts

```diff
@@ -0,0 +1,222 @@
+import {ArgumentType, BaseCommand, ICommandContext, ICommandInfo} from "../BaseCommand";
```

Comment:
Crash when I attempt to remove one of my names.  Again, similar to issue with AddorChangeNames. 
![image](https://user-images.githubusercontent.com/97406520/149033556-b2c29fd7-c220-4a12-bf9b-ea40019d7e49.png)
![image](https://user-images.githubusercontent.com/97406520/149033563-1565f82d-d1f6-441c-8408-75dcb44981fb.png)

Reply:
Works as intended for example account.

Reply:
Again, weird issue with Beep.
![image](https://user-images.githubusercontent.com/97406520/149034169-332b8e54-13ea-4de4-8035-7306d797bfe9.png)